includes/Modules/Thank_With_Google/Supporter_Wall_Widget.php

```diff
@@ -43,6 +43,13 @@ public function __construct() {
 	 * @param array $instance The widget instance.
 	 */
 	public function form( $instance ) {
+		$instance = wp_parse_args(
+			$instance,
+			array(
+				'title' => __( 'Thank with Google Supporters', 'google-site-kit' ),
+			)
+		);
+
 		$title    = ! empty( $instance['title'] ) ? $instance['title'] : '';
```

Comment:
This line can be simplified now that `title` will always exist in the resulting `instance` array.
```suggestion
		$title    = $instance['title'];
```